arithmetization/src/main/java/net/consensys/linea/zktracer/ZkTracer.java

```diff
@@ -53,11 +55,21 @@ public class ZkTracer implements ConflationAwareOperationTracer {
   /** The {@link GasCalculator} used in this version of the arithmetization */
   public static final GasCalculator gasCalculator = new LondonGasCalculator();
 
-  @Getter private final Hub hub = new Hub();
+  @Getter private final Hub hub;
   private final Map<String, Integer> spillings = new HashMap<>();
   private Hash hashOfLastTransactionTraced = Hash.EMPTY;
 
   public ZkTracer() {
+    this(
+        LineaL1L2BridgeConfiguration.builder()
+            .contract(Address.fromHexString("0xDEAD"))
+            .topic(Bytes.fromHexString("0xBEEF"))
+            .build());
+  }
```

Comment:
this should not be needed anymore

```suggestion
```

Reply:
This is still used by all the places where the L2L1 bridge settings are useless, e.g. in the tracing endpoints, the continuous tracer, etc.

Reply:
replaced with a constant `LineaL1L2BridgeConfiguration.EMPTY`